File: src/main/Java/boj_step4/boj10807.java
import java.io.*;
import java.util.StringTokenizer;

public class boj10807 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int size = Integer.parseInt(reader.readLine());
        int[] list = new int[size];
        StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
        int key = Integer.parseInt(reader.readLine());
        int count = 0;
        for (int i = 0; i < size; i++) {
            list[i] = Integer.parseInt(tokenizer.nextToken());
            if (list[i] == key) count++;
        }
        writer.append(String.valueOf(count));
        writer.close();
    }
}

File: src/main/kotlin/boj10807.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    var n = r.readLine()!!.toInt()
    val arr = r.readLine()!!.split(" ").map { it.toInt() }
    val v = r.readLine()!!.toInt()
    var cnt = 0
    for (i in 0 until n) if(arr[i] == v) cnt ++
    w.write(cnt.toString())
    w.close()
}